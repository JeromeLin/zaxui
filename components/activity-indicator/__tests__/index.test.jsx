import React from 'react';
import { render } from 'enzyme';
import toJson from 'enzyme-to-json';
import ActivityIndicator from '../index';

describe('ActivityIndicator', () => {
  it('renders correctly', () => {
    const wrapper = render(<ActivityIndicator />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
