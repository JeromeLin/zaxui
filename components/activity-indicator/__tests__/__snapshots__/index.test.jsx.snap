// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityIndicator renders correctly 1`] = `
<svg
  class="za-activity-indicator theme-primary"
  viewBox="0 0 62 62"
>
  <circle
    class="za-activity-indicator-path"
    cx="31"
    cy="31"
    fill="none"
    r="28.5"
    style="stroke-width:5"
  />
  <circle
    class="za-activity-indicator-line"
    cx="31"
    cy="31"
    fill="none"
    r="28.5"
    style="stroke-dasharray:26.860617188192727 179.0707812546182;stroke-width:5"
  />
</svg>
`;
